src/development/cli/api-tokens.md

```diff
@@ -22,26 +22,29 @@ Now set that token to an environment variable named `PLATFORMSH_CLI_TOKEN` on yo
 >
 > If running CLI commands from any automated system, including a Platform.sh cron task, we urge you to use the `--no-wait` flag on any commands that may take more than a second or two to avoid blocking the process.
 
-## Automation users
+## Machine users
+
+For security reasons we recommend to create a dedicated machine user to run automation tasks such as taking backups, renewing SSL certificates or triggering source operations. We also recommend to create a unique machine user for each project to be automated.
```

Comment:
"We recommend creating".  "We recommend to create" is a very French accent. :smile:

Reply:
(Appears twice.)